### ISSUE TITLE
Standard reply box gets replaced when you want to send a nonenrypted email.

### ISSUE THREAD

**Issue by michael-volynets** [FlowCrypt/flowcrypt-browser#1735]
**editted by Tom with latest issue description to avoid confusion**
----------------------

Existing behavior to preserve:
- if last message is encrypted, and I click encrypted reply, render encrypted reply box
- if last message is plain, and I click plain reply, render plain reply
- if last message is plain, and I click encrypted reply, render encrypted reply

Current unwanted behavior:
 - if last message is encrypted, and I **click plain reply icon on top right**, it still shows an encrypted reply box

Instead, we should find a way to NOT replace the plain reply box that shows **after explicitly clicking on plain reply** and instead augument it with a very bright red box above it, which says: `The last message was encrypted, but you are composing a reply without encryption. [switch to encrypted reply]`

In a separate issue, we should have a way to switch from within the secure reply to replying without encryption.

That should be less invasive and make everyone happy.

**Comment by tomholub** [comment]
This is historically intended behavior, but should improve it.

I will add some notes here over the weekend.

**Comment by tomholub** [comment]
- add info about reply buttons in general

**Comment by tomholub** [comment]
#920 is somewhat affecting it, once that one lands, I will re-visit this issue

**Comment by tomholub** [comment]
I'm not entirely sure what is the right approach here.

Users will habitually be clicking the original reply button and think that the reply will be encrypted, but it won't.

At the same time, more mindful users will sometimes want to actually reply without encryption, but we don't allow them atm.

Best would be if it's possible to show a warning, something like:

> The message you are replying to is encrypted. Do you really want to reply without encryption?

But I'm really not sure.

**Comment by tomholub** [comment]
Moving to discussion, curious about your input

**Comment by tomholub** [comment]
Thought about this some more.

Existing behavior to preserve:
- if last message is encrypted, and I click encrypted reply, render encrypted reply box
- if last message is plain, and I click plain reply, render plain reply
- if last message is plain, and I click encrypted reply, render encrypted reply

Current unwanted behavior:
 - if last message is encrypted, and I **click plain reply icon on top right**, it still shows an encrypted reply box

Instead, we should find a way to NOT replace the plain reply box that shows **after explicitly clicking on plain reply** and instead augument it with a very bright red box above it, which says: `The last message was encrypted, but you are composing a reply without encryption. [switch to encrypted reply]`

In a separate issue, we should have a way to switch from within the secure reply to replying without encryption.

That should be less invasive and make everyone happy.

**Comment by limonte** [comment]
@tomholub a small detail to clarify here:

Given that the user has opened the plain message:

![image](https://user-images.githubusercontent.com/6059356/75777374-8cdfaf00-5d5e-11ea-9d84-2e5850c4a4f3.png)

When the user clicks the Secure Reply button ![image](https://user-images.githubusercontent.com/6059356/75777257-5144e500-5d5e-11ea-9262-d9fa6843713f.png)
And the user clicks the Delete Draft button ![image](https://user-images.githubusercontent.com/6059356/75777333-79344880-5d5e-11ea-9449-ca298caae439.png)

Currently, the user is seeing this:

![image](https://user-images.githubusercontent.com/6059356/75777418-9ff27f00-5d5e-11ea-9c44-500fc21debab.png)

Which is different from the initial UI:

![image](https://user-images.githubusercontent.com/6059356/75777374-8cdfaf00-5d5e-11ea-9d84-2e5850c4a4f3.png)

Do I understand correctly that the desired would be to show the initial UI after deleting a draft?

**Comment by tomholub** [comment]
That may be too difficult to achieve. But if you can do it, sure, I guess that would be preferable.
